fontforge/lookups.c

```diff
@@ -1319,17 +1319,22 @@ static struct {
     const char *text;
     uint32 tag;
 } localscripts[] = {
+    { N_("Adlam"), CHR('a','d','l','m') },
+    { N_("Ahom"), CHR('a','h','o','m') },
+    { N_("Anatolian Hieroglyphs"), CHR('h','l','u','w') },
 /* GT: See the long comment at "Property|New" */
 /* GT: The msgstr should contain a translation of "Arabic", ignore "Script|" */
```

Comment:
I *think* this comment meant to be on the first line, and Arabic is used as example because it happens to be the first one.

Reply:
The `GT:` comment is associated in the .po files with the immediately following string, so it should precede the first entry that uses a vertical bar. Here is an existing example:
https://github.com/fontforge/fontforge/blob/b6a4cb45eb9234cfd3235dab1513cb624073226d/fontforgeexe/lookupui.c#L304-L316

Reply:
Right, I missed that. Sorry for the noise.